ReactWindows/ReactNative.Net46/Views/Image/ReactImageManager.cs

```diff
@@ -295,9 +296,11 @@ private void SetUriFromSingleSource(Border view, string source)
                     status => OnImageStatusUpdate(view, status),
                     _ => OnImageFailed(view));
 
-                using (var stream = BitmapImageHelpers.GetStreamAsync(source))
+                using (Stream stream = BitmapImageHelpers.GetStreamAsync(source))
```

Comment:
Any reason to not use var? Also could remove `using`  if you did change this back.